src/input/keyboard.js

```diff
@@ -226,6 +232,17 @@ class Keyboard extends EventHandler {
         }
     }
 
+    _handleVisibilityChange() {
+        if (document.visibilityState === 'hidden') {
+            this._handleWindowBlur();
+        }
+    }
+
+    _handleWindowBlur() {
+        this._keymap = {};
+        this._lastmap = {};
+    }
```

Comment:
Will we need to fire keyup events so that the application can handle keys being 'released' by window blur?

Reply:
I thought about it. But we won't be able to do that without faking Event object.
In the web, keydown > keyup - is not a guaranteed flow, so we probably should not change or try to mitigate it, as there will be a lot of edge cases.

This PR only focuses on static access to key states, not events. That would need to be a separate PR.

Reply:
Right, I see what you mean